src/veritysetup/veritysetup.c

```diff
@@ -64,6 +76,25 @@ static int save_roothashsig_option(const char *option, bool strict) {
                                "base64 string encoding signature prefixed by base64:.");
 }
 
+static int parse_block_size(const char *t, uint64_t *size) {
+        uint64_t u;
+        int r;
+
+        r = parse_size(t, 1024, &u);
+        if (r < 0)
+                return r;
+
+        if (u < 512 || u > (512 * 1024))
+                return -ERANGE;
+
+        if ((u % 512) != 0 || !CONST_ISPOWEROF2(u))
```

Comment:
this is not const, just use regular ISPOWEROF2() instead of CONST_ISPOWEROF2()

Reply:
done